packages/react/menu/src/Menu.tsx

```diff
@@ -91,12 +91,12 @@ interface MenuProps {
 }
 
 const Menu: React.FC<MenuProps> = (props: ScopedProps<MenuProps>) => {
-  const { __scopeMenu, open = false, children, onOpenChange, modal = true } = props;
+  const { __scopeMenu, open = false, children, dir, onOpenChange, modal = true } = props;
```

Comment:
What do you think about doing `dir = 'ltr'` here rather than having the double `||` later?